doc/api/next_api_changes/2018-10-23-AL.rst

```diff
@@ -0,0 +1,12 @@
+``Cn`` colors now support ``n>=10``
+```````````````````````````````````
+
+It is now possible to go beyond the tenth color in the property cycle using
+``Cn`` syntax, e.g. ``plt.plot([1, 2], color="C11")`` now uses the 11th color
```

Comment:
`C11` should be the 12th color, right?

In general what happens if you specify `C10` on a cycler with 10 colors? Is it ensured that it will result in the first color then?

Reply:
Depends on whether you think it starts at zeroth or first.  I think matching the idex is easier, if not necessarily correct in English.
In any case I think the meaning is clear...

Reply:
Ähm???? Sorry but `a[1]` takes the **second** element of `a`, not the first. That has nothing to do with language.

Reply:
Sure, changed.